packages/vue-instantsearch/src/components/DynamicWidgets.js

```diff
@@ -65,6 +65,13 @@ export default {
       type: Number,
       default: undefined,
     },
+    // TODO: this only adds the dynamic widgets to the parent index, not the
+    // widgets that it renders. The parentIndexId should be forwarded.
+    parentIndexId: {
+      type: String,
+      required: false,
+      default: undefined,
+    },
```

Comment:
this should be figured out if we want to have parentIndexId on every widget

Reply:
maybe we could write to the vnode before rendering so it has the extra parentIndexId prop, but that seems weird.